apps/prairielearn/elements/pl-excalidraw/info.json

```diff
@@ -0,0 +1,17 @@
+{
+  "controller": "pl-excalidraw.py",
```

Comment:
Is the name `pl-excalidraw` appropriate? 

Arguments for

**Yes**

- We don't want to call it `pl-sketch` because we might include a raster drawing widget in the future
- Easy discoverability - on looking at the element name pl-excalidraw, someone could search for _excalidraw_, compared to a more generic name.

**No**

- Excalidraw is an implementation detail
- Poor discoverability - someone looking at the list of elements may not know what pl-excalidraw stands for

Reply:
I think `pl-excalidraw` is probably our best option. Excalidraw is only _sort of_ an implementation detail, that is, it's Excalidraw's document format that will show up in submissions, and it's Excalidraw's document format that instructors must use when providing an answer file. Practically speaking, we'll never be able to swap out another drawing engine in the future. I say we embrace the fact that we're using Excalidraw and keep it in the name.

Reply:
resolving!